man/journal-upload.conf.xml

```diff
@@ -60,6 +60,26 @@
         <xi:include href="version-info.xml" xpointer="v232"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Compression=</varname></term>
+
+        <listitem><para>Compression algorithm to be applied to logs data before sending.
+        Supported values are <literal>zstd</literal>, <literal>xz</literal>, <literal>lz4</literal>, <literal>none</literal>.
+        Defaults to <literal>none</literal>.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>CompressionLevel=</varname></term>
+
+        <listitem><para>Specifies the compression level of data to be uploaded.
+        Each algorithm has it's own range of allowed integer values:  xz - [1 - 9], zstd - [1 - 22], lz4 - [1 - 9].
+        Defaults to -1, and the default compression level for each compression algorithm will be used.</para>
```

Comment:
Maybe, better to not mention the detailed range, but just say this takes an integer and acceptable range depends on the algorithm. Also, please drop the internal default `-1`, but say `Defaults to unset, ...`.

Please mention that this setting will be ignored when `Compression=none`.